src/Sylius/Bundle/PromotionBundle/Resources/translations/flashes.en.yml

```
@@ -6,3 +6,5 @@ sylius:
         generate: 'Promotion coupons have been successfully generated.'
     catalog_promotion:
         remove: 'Removing of catalog promotion has been requested. This process can take a while depending on the number of affected products.'
+        archive: 'Archiving of catalog promotion has been requested. This process can take a while depending on the number of affected products.'
```

I see that new translation keys are in line with `flashes.en.yml`, but I personally would have preferred.

```suggestion
        archival_in_progress: 'Archiving of catalog promotion has been requested. This process can take a while depending on the number of affected products.'
```